### ISSUE TITLE
AQ Data Landscape

### ISSUE THREAD

**Issue by olafveerman** [openaq/openaq.org#374]
**tl;dr** This issue explores how we can integrate the research results of the **AQ Data Landscape** into the website. [This prototype](http://openaq-landscape.surge.sh/#/landscape) is a first exploration of how we can make the great research that the team is doing more accessible.

---

## Goal
The goal of developing an interface for the Landscape is to make the results more easily accessible for the general public, and give it a more polished look and feel. The current spreadsheet provides a low barrier to edit and contribute, but isn't the most pleasant way to quickly get insights.

## Implementation details

* The prototype (PR: https://github.com/openaq/openaq.org/pull/373) is pulling directly from a Google Spreadsheet. This means we can integrate the ranking without having to change the current data collection approach. It keeps a high level of transparency, and a low barrier to contribute for the community. I imagine that we include plenty of CTAs on the page to improve the data.
* The details tab of each country (under View More) can scale and grow. I didn't put any thought into the content and design of it, but we have a lot of space to add extra information here.

## Open questions
The prototype still needs a lot of UI and UX polish, and improvements to the markup. Before we do this, there are a number of methodological questions to answer:

* some countries publish data at the regional level (eg. Australia, Brazil). We can add those regional details under the View More tab, but how do we roll this up into the country level summary (the four ticks).
* at the moment, there is no indication of the amount of stations (per capita / area). Something like this would allow us to add more nuance, beyond the 4 ticks.
* do we want to make periodic updates to the landscape (eg. every 3 / 6 months), or do we want to do rolling updates? In case it's the latter, we can record 'updated at' for every country.
* what other content do we want to present on this page? An intro into the overall state of AQ? The report?

## Next steps

* answer the open questions, and do a proper review of the content
* design & implement

**Comment by RocketD0g** [comment]
This is awesome. 

Some thoughts inline below on the open questions above.

> ## Open questions
> The prototype still needs a lot of UI and UX polish, and improvements to the markup. Before we do this, there are a number of methodological questions to answer:
> 
> * some countries publish data at the regional level (eg. Australia, Brazil). We can add those regional details under the View More tab, but how do we roll this up into the country level summary (the four ticks).

Perhaps checks get a different color if a country only has regional data in the first place? Like yellow? 

I will also flag that for the first iteration of this, it may be too time-intensive to go through each source and determine whether it is only regional right now.

> * at the moment, there is no indication of the amount of stations (per capita / area). Something like this would allow us to add more nuance, beyond the 4 ticks.

I wonder if we could pull this in from OpenAQ directly? This *would not* represent the total possible # of stations that may exist in a country, of course -just what is appearing on OpenAQ. We'd have to be thoughtful in messaging with that.

> * do we want to make periodic updates to the landscape (eg. every 3 / 6 months), or do we want to do rolling updates? In case it's the latter, we can record 'updated at' for every country.

I imagine, ideally, we update this on a rolling basis, so perhaps the "updated at' for each country? 

> * what other content do we want to present on this page? An intro into the overall state of AQ? The report?
>

The report seems like a good idea, as well as some background/intro content, and a CTA for corrections/omissions. 

It would be kind of cool for countries that don't have any data source on OpenAQ - or perhaps an issue, if there was a CTA to create a GH issue or point to the existing one.

Other thoughts/reactions from folks? 
Tagging @AreteY and @sruti for any thoughts, too.

Thanks again for doing this, Olaf.
